lib/Common/DataStructures/List.h

```diff
@@ -360,6 +360,7 @@ namespace JsUtil
         void SetItem(int index, const T& item)
         {
             EnsureArray(index + 1);
+            // TODO: (SWB)(leish) find a way to force user defined copy constructor
```

Comment:
>// TODO: (SWB)(leish) find a way to force user defined copy constructor [](start = 12, length = 71)

This looks like "operator =", why copy constructor is involved? Could you please elaborate (possibly also update the comment)?